ios/Podfile

```diff
@@ -35,21 +35,21 @@ target 'SSIWallet' do
 #    pod 'Permission-BluetoothPeripheral', :path => "#{permissions_path}/BluetoothPeripheral"
```

Comment:
would it make sense to just remove the ones we do not need?

Reply:
Yes, but at the time I had no idea which ones we are actually using in the app, still don't actually. I just commented out as many as possible so Apple would stop whining. Waiting for feedback whether the app is fully functional, maybe some have to be enabled still.